src/PhpSpreadsheet/Cell/AddressHelper.php

```diff
@@ -7,6 +7,14 @@
 class AddressHelper
 {
     public const R1C1_COORDINATE_REGEX = '/(R((?:\[-?\d*\])|(?:\d*))?)(C((?:\[-?\d*\])|(?:\d*))?)/i';
+    // I can't find complete documentation for this. Using:
+    // https://answers.microsoft.com/en-us/officeinsider/forum/all/indirect-function-is-broken-at-least-for-excel-in/1fcbcf20-a103-4172-abf1-2c0dfe848e60
+    private const R1C1_INTERNATIONALIZATION_REGEXES = [
+        '/^(R(\[?[-+]?\d*\]?))(C(\[?[-+]?\d*\]?))$/i', // English
```

Comment:
Okay, assuming you are right about the first character of ROW/COLUMN being used (it certainly checks with the known examples), that won't be difficult to recode, but it'll probably take a couple of days to redesign the unit tests. I think (not certain) that I can verify a live spreadsheet in French, but could definitely use your help with Cyrillic.

Reply:
Yes, I can switch to and from French.

Reply:
Helpful, not! MS Excel's built in formula translator doesn't work with R1C1 format at all :-( So I'll need to check on Cyrillic the hard way

Reply:
https://www.rondebruin.nl/win/s9/win013.htm has an INDIRECT example which says that Russian uses R and C, which is probably just as well because the Russian translations for ROW and COLUMN begin with the same letter. For now, I will tailor my solution to use the first letter of translated ROW/COLUMN if they (a) are different from each other and (b) are both in the ASCII range, otherwise R/C. I will push an update later today.

Reply:
Possible non-Cyrillic problems for languages for which we have translations are Turkish (ROW/COLUMN begin with same letter), Czech (ROW begins with an accented letter), Polish (COLUMN translated as NR.KOLUMNY - use N or K?).

Reply:
New push is ready for review.

Reply:
@MarkBaker My latest commit [14c8842](https://github.com/PHPOffice/PhpSpreadsheet/pull/3052/commits/14c8842b50450b84aecf165ba1520a99c4df7195) now makes the change entirely dependent on the locale spreadsheet, which I think is finally the correct approach (uncertainty about that is what's kept me from merging till now). See commit message for details. Marking this conversation as resolved, intending to merge the change within a few days unless you want other changes, or want to study other locales.